src/FormMigration/FormMetaDecorator.php

```diff
@@ -446,4 +447,73 @@ public function getFormFieldsPlacement(): string
     {
         return give_get_meta($this->form->id, '_give_ffm_placement', true);
     }
+
+    /**
+     * @unreleased
+     */
+    public function getFeeRecoverySettings(): array
+    {
+        $feeRecoveryStatus = $this->getMeta('_form_give_fee_recovery');
+
+        if ($feeRecoveryStatus === 'disabled') {
+            return [];
+        }
+
+        if ($feeRecoveryStatus === 'global') {
+            return [
+                'useGlobalSettings' => true,
+            ];
+        }
+
```

Comment:
Just add one more conditional here for:

```
if ($feeRecoveryStatus !== 'enabled') {
      return [];
}
```

I've come to realize its better to assume legacy meta is either empty or disabled by default - so we should check if its specifically enabled instead of assuming it is because it's _not_ disabled.

Reply:
Resolved here: af0ff4719c21c21d87b3d7b2831633795d01faee